docs/rules/sort-keys.md

```diff
@@ -53,7 +53,7 @@ let obj = {a: 1, ["c" + "d"]: 3, b: 2};
 let obj = {a: 1, [`${c}`]: 3, b: 2};
 let obj = {a: 1, [tag`c`]: 3, b: 2};
 
-// This rule ignores objects that have a spread operator in them.
+// This rule ignores objects that have a non-boundary spread operator in them.
```

Comment:
I think this comment has the potential to be confusing because the user might not understand what a "boundary" spread operator is. Additionally, it seems like the comment doesn't quite describe the behavior of the rule; for example, the rule will still report an error in this case, even though `...c` is a "non-boundary" spread:

```js
let obj = {b: 1, a: 2, ...c, d: 3};
```

Maybe this could be reworded as something like this, although feel free to use a different wording if you think you can make it clearer:

> This rule does not report unsorted properties that are separated by a spread property.